pyramid/tests/test_authentication.py

```diff
@@ -758,6 +767,12 @@ def test_identify_cookie_timed_out(self):
         result = helper.identify(request)
         self.assertEqual(result, None)
 
+    def test_identify_cookie_str_timeout(self):
```

Comment:
How does this test the timeout of the cookie? A bogus `auth_tkt` is already invalid and will always "timeout".